src/registrar/templates/application_review.html

```diff
@@ -20,108 +20,158 @@ <h1> Review and submit your domain request </h1>
+          {% endwith %}
+        {% endif %}
+
+        {% if step == Step.AUTHORIZING_OFFICIAL %}
+          {% namespaced_url 'application' step as application_url %}
+          {% if application.authorizing_official is not None %}
+            {% with title=form_titles|get_item:step value=application.authorizing_official %}
+              {% include "includes/summary_item.html" with title=title value=value heading_level=heading_level editable=True edit_link=application_url contact='true' %}
+            {% endwith %}
+          {% else %}
+            {% with title=form_titles|get_item:step value="Incomplete" %}
+              {% include "includes/summary_item.html" with title=title value=value heading_level=heading_level editable=True edit_link=application_url %}
+            {% endwith %}
           {% endif %}
-          {% if step == Step.CURRENT_SITES %}
-            <ul class="add-list-reset">
-              {% for site in application.current_websites.all %}
-                <li>{{ site.website }}</li>
-              {% empty %}
-                <li>None</li>
-              {% endfor %}
-            </ul>
+        {% endif %}
+
+        {% if step == Step.CURRENT_SITES %}
+          {% namespaced_url 'application' step as application_url %}
+          {% if application.current_websites.all %}
+            {% with title=form_titles|get_item:step value=application.current_websites.all %}
+              {% include "includes/summary_item.html" with title=title value=value heading_level=heading_level editable=True edit_link=application_url list='true' %}
+            {% endwith %}
+          {% else %}
+            {% with title=form_titles|get_item:step value='None' %}
+              {% include "includes/summary_item.html" with title=title value=value heading_level=heading_level editable=True edit_link=application_url %}
+            {% endwith %}
           {% endif %}
-          {% if step == Step.DOTGOV_DOMAIN %}
-            <ul class="add-list-reset margin-bottom-105">
-              <li>{{ application.requested_domain.name|default:"Incomplete" }}</li>
-            </ul>
-            <ul class="add-list-reset">
+        {% endif %}
+
+        {% if step == Step.DOTGOV_DOMAIN %}
+          {% namespaced_url 'application' step as application_url %}
+          {% with title=form_titles|get_item:step value=application.requested_domain.name|default:"Incomplete" %}
+            {% include "includes/summary_item.html" with title=title value=value heading_level=heading_level editable=True edit_link=application_url %}
+          {% endwith %}
+
+          {% if application.alternative_domains.all %}
+            <h3 class="margin-bottom-0 text-base-darker">Alternative domains</h3>
+            <ul class="usa-list margin-top-0">
               {% for site in application.alternative_domains.all %}
                 <li>{{ site.website }}</li>
               {% endfor %}
             </ul>
```

Comment:
(Comment) This seems similar in principle to the other summary items, but with a header and slightly different classes. Its not something you should address in this PR (Too much effort for too little reward) but I think summary_item could be genericized enough to handle this edge case

Reply:
I agree.  We considered this.  But left this edge case outside of summary_item for now.  Might be worth revisiting this decision at a later date.